docs/env.variables.md

```diff
@@ -45,3 +45,11 @@ $ VERDACCIO_FORWARDED_PROTO=CloudFront-Forwarded-Proto verdaccio --listen 5000
 #### VERDACCIO_STORAGE_PATH
 
 By default, the storage is taken from config file, but using this variable allows to set it from environment variable.
+
+#### VERDACCIO_LOGGER_COLORS
```

Comment:
Please, prefix with `EXPERIMENTAL__´, I not sure if this will be moved to 6.x, I would like extract the prettifier as a plugin

Reply:
roger that